prototype/MsiV2DemoApp/Program.cs

```diff
@@ -0,0 +1,806 @@
+﻿// Managed Identity + mTLS PoP + mTLS resource call (Console Demo)
+// .NET 8 
+//
+// Menu:
+//   Acquire Tokens
+//     1   - SAMI → Token from IDP (force refresh)
+//     1a  - SAMI → Token from Cache
+//     2   - UAMI → Token from IDP (force refresh)
+//     2a  - UAMI → Token from Cache
+//   Call Resource
+//     3   - SAMI token + cert → Call resource (mTLS)
+//     4   - UAMI token + cert → Call resource (mTLS)
+//   Display & Toggles
+//     F   - Toggle Full Token view (ON/OFF)

```

Comment:
Consider adding a warning/reminder about safe handling of full token.

Reply:
added